frontend/src/metabase/search/utils/user-search-params/index.ts

```diff
@@ -0,0 +1 @@
+export { parseUserIdString, convertUserIdToString } from "./user-search-params";
```

Comment:
`frontend/src/metabase/search/utils/index.ts` exports everything from `./search-location` but not `./user-search-params` and not `./enabled-search-type` - let's make this consistent?

(I suggest to avoid intimate imports, i.e. follow what's been done with `./search-location`).

Reply:
Looks like that got lost in a merge conflict that I was dealing with - I've changed that